specification/maps/resource-manager/Microsoft.Maps/preview/2023-12-01-preview/maps-management.json

```diff
@@ -778,10 +736,271 @@
           "default": {
             "description": "An unexpected error occurred.",
             "schema": {
-              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Maps/accounts/{accountName}/privateLinkResources": {
```

Comment:
Point-Get is also needed: `"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Kusto/clusters/{clusterName}/privateLinkResources/{privateLinkResourceName}"`

Reply:
@ms-zhenhua we submitted a PR on the dev branch, can you please take a look and approve, it looks like status checks are not running after trying `/azp run` multiple times. https://github.com/Azure/azure-rest-api-specs/pull/27056